src/api/writer-generator/python/python.ts

```diff
@@ -0,0 +1,711 @@
+        }
+    }
+
+    private generateDefaultImports(): void {
+        this.pyImportFrom("__future__", "annotations");
+        this.pyImportFrom("pydantic", "BaseModel", "ConfigDict", "Field", "PositiveInt");
+        this.pyImportFrom("typing", "List as PyList", "Literal");
+    }
+
+    private generateDependenciesImports(schema: RegularTypeSchema): void {
+        if (!schema.dependencies || schema.dependencies.length === 0) return;
+
+        this.importComplexTypeDependencies(schema.dependencies);
+        this.importResourceDependencies(schema.dependencies);
+    }
+
+    private importComplexTypeDependencies(dependencies: Identifier[]): void {
+        const complexTypeDeps = dependencies.filter((dep) => dep.kind === "complex-type");
+        const depsByPackage = this.groupDependenciesByPackage(complexTypeDeps);
+
+        for (const [pyPackage, names] of Object.entries(depsByPackage)) {
+            this.pyImportFrom(pyPackage, ...names.sort());
+        }
+    }
+
+    private importResourceDependencies(dependencies: Identifier[]): void {
+        const resourceDeps = dependencies.filter((dep) => dep.kind === "resource");
+
+        for (const dep of resourceDeps) {
+            this.pyImportType(dep);
+
+            const familyName = `${pascalCase(dep.name)}Family`;
+            const familyPackage = `${this.pyFhirPackage(dep)}.resource_families`;
+            this.pyImportFrom(familyPackage, familyName);
+        }
+    }
+
+    private groupDependenciesByPackage(dependencies: Identifier[]): ImportGroup {
+        const grouped: ImportGroup = {};
+
+        for (const dep of dependencies) {
+            const pyPackage = this.pyPackage(dep);
+            if (!grouped[pyPackage]) {
+                grouped[pyPackage] = [];
+            }
+            grouped[pyPackage].push(dep.name);
+        }
+
+        return grouped;
+    }
+
+    pyImportFrom(pyPackage: string, ...entities: string[]): void {
+        const oneLine = `from ${pyPackage} import ${entities.join(", ")}`;
+
+        if (this.shouldUseSingleLineImport(oneLine, entities)) {
+            this.line(oneLine);
+        } else {
+            this.writeMultiLineImport(pyPackage, entities);
+        }
+    }
+
+    private shouldUseSingleLineImport(oneLine: string, entities: string[]): boolean {
+        return oneLine.length <= MAX_IMPORT_LINE_LENGTH || entities.length === 1;
+    }
+
+    private writeMultiLineImport(pyPackage: string, entities: string[]): void {
+        this.line(`from ${pyPackage} import (\\`);
+        this.indentBlock(() => {
+            const remaining = [...entities];
+            while (remaining.length > 0) {
+                const line = this.buildImportLine(remaining);
+                this.line(line);
+            }
+        });
+        this.line(")");
+    }
+
+    private buildImportLine(remaining: string[]): string {
+        let line = "";
+
+        while (remaining.length > 0 && line.length < MAX_IMPORT_LINE_LENGTH) {
```

Comment:
Maybe we can add it as a helper to Writer? E.g.: `this.line(TEXT, maxWidth=80, splitOn=/ ,/)`?